library/alloc/src/lib.rs

```diff
@@ -144,6 +144,7 @@
 #![feature(unicode_internals)]
 #![feature(unsize)]
 #![feature(std_internals)]
+#![feature(unicode_converter)]
```

Comment:
This feature name is too generic, perhaps something more like `#![feature(str_chars_casemapped)]`?